src/backend/api/Fusion.Resources.Database/Entities/DbContract.cs

```diff
@@ -15,6 +16,14 @@ public class DbContract
         public DateTimeOffset Allocated { get; set; }
         public DbPerson AllocatedBy { get; set; } = null!;
         public Guid AllocatedById { get; set; }
-    }
 
+        public string CompanyName { get; set; } = null!;
```

Comment:
Company name would not be sufficient.. I suggest doing a lookup to the contract using the resolver, on the fly.

Reply:
Ok, will change it.